application/gui/input_dialog.py

```diff
@@ -17,6 +17,8 @@ class OverwriteOption:
 	NEW_NAME = 1
 	APPLY_TO_ALL = 2
 
+# header bars are only available in GTK 3.12 and up
+_use_header_bar = { 'use_header_bar': True } if hasattr(Gtk.Dialog, 'get_header_bar') else {}
```

Comment:
Reworked pushed - is that what you had in mind?

On Mon, May 2, 2016 at 11:52 AM, Mladen Mijatov <notifications@github.com>
wrote:

> In application/gui/input_dialog.py
> <https://github.com/MeanEYE/Sunflower/pull/227#discussion_r61725486>:
>
> > @@ -17,6 +17,8 @@ class OverwriteOption:
> >  	NEW_NAME = 1
> >  	APPLY_TO_ALL = 2
> >
> > +# header bars are only available in GTK 3.12 and up
> > +_use_header_bar = { 'use_header_bar': True } if hasattr(Gtk.Dialog, 'get_header_bar') else {}
>
> Move this to InputDialog as class property and make it "public" (
> use_header_bar). This way it will make sense to use it in all dialogs. If
> dialog is not inheriting InputDialog it's still simple enough to use
> InputDialog.use_header_bar as it's class property.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MeanEYE/Sunflower/pull/227/files/a0a15037a218ea668adc86385ea26f73ab218406#r61725486>
>